index.html

```diff
@@ -0,0 +1,256 @@
+      <h2>Main Program</h2>
+      <hr class="mx-3"/>
+      <div class="program-grid">
+        <div>
+          <img src="assets/images/cp/program_icon_01.png" alt="Lecture" srcset="">
+          <h5>Lecture</h5>
+          <p>Listen to the speakers from various countries share their thoughts on industry standard practices.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_02.png" alt="CC Exhibition" srcset="">
+          <h5>CC Exhibition</h5>
+          <p>Get a chance to get your various creations/designs  displayed in our website for potential employees.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_03.png" alt="Forum" srcset="">
+          <h5>Forum</h5>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_04.png" alt="Workshop" srcset="">
+          <h5>Workshop</h5>
+          <p>Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_05.png" alt="CC Ignite" srcset="">
+          <h5>CC Ignite</h5>
+          <p>Get opportunities to network With other up-comming devs around the world.</p>
+        </div>
+      </div>
+      <div class="d-flex center">
+        <a href="#" class="desktop see-whole">SEE THE WHOLE PROGRAM</a>
+        <button class="mobile join">Join CC Global Summit 2023</button>
+      </div>
+    </section>
+    <section id="speakers">
+      <h2>
+        Featured Tutors
+      </h2>
+      <hr  class="mx-3 mb-2"/>
+      <div class="d-flex speaker-group">
+        <div class="speaker-block">
+          <div class="speaker-image">
+            <img src="assets/images/Speakers/speaker_01.png" alt="">
+          </div>
+          <div class="speaker-desc">
+            <h4>Yochai Benkler</h4>
+            <p class="text-red">Berkman professor of Enterprenural Legal Studies at Harvard Law School.</p>
+            <p>Benkler studies commons-based peer production, and published his seminal book The Wealth of Networks in 2006</p>
+          </div>
+        </div>
+        <div class="speaker-block">
+          <div class="speaker-image">
+            <img src="assets/images/Speakers/speaker_01.png" alt="">
+          </div>
+          <div class="speaker-desc">
+            <h4>Yochai Benkler</h4>
+            <p class="text-red">Berkman professor of Enterprenural Legal Studies at Harvard Law School.</p>
+            <p>Benkler studies commons-based peer production, and published his seminal book The Wealth of Networks in 2006</p>
+          </div>
+        </div>
+        <div class="speaker-block">
+          <div class="speaker-image">
+            <img src="assets/images/Speakers/speaker_01.png" alt="">
+          </div>
+          <div class="speaker-desc">
+            <h4>Yochai Benkler</h4>
```

Comment:
- [ ] [OPTIONAL] I am only marking this as optional because the previous reviewer did not point it out but I strongly suggest you implement it. It is required that the featured speakers section is generated dynamically. Please kindly implement this.